src/scripts/main.js

```diff
@@ -1 +1,21 @@
 'use strict';
+
+window.addEventListener('DOMContentLoaded', () => {
+  const largeImg = document.querySelector('#largeImg');
+  const imgList = document.querySelector('.gallery__list');
+
+  imgList.addEventListener('click', (e) => {
+    e.preventDefault();
+
+    const target = e.target;
```

Comment:
You can use Destructuring assignment her
```suggestion
    const {target} = e;
```